tests/Benchmarks/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿using BenchmarkDotNet.Running;
+
+namespace Benchmarks
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+            => BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
```

Comment:
Is there a way to run the tests on both CLR and Core without changing the sources? I think it would be interesting to be able to run some CLR-Core comparisons for APIs that use fast and slow span.

Reply:
And more generally, it would be useful to add some comment to this Main explaining what command line args can be passed (maybe simple inline, and then link to a BDN docs).

Reply:
We should also add a README.md summarizing how we can run the tests and common args that are useful here (similar to what we had before https://github.com/dotnet/corefxlab/blob/master/scripts/PerfHarness/README.md). That doc can evolve as we do the cleanup.

Reply:
> Is there a way to run the tests on both CLR and Core without changing the sources? 

not yet. I can add sample config to the code and add comment for how to use it

Unfortunately I don't have time to add the docs now. I will serarch for some `.md` file with docs that we could reuse.